db/migrate/20151119100304_update_blank_content_item_content_ids.rb

```diff
@@ -0,0 +1,33 @@
+class UpdateBlankContentItemContentIds < Mongoid::Migration
+  def self.up
+    content_items = []
+
+    ContentItem.where(content_id: nil).each do |item|
```

Comment:
Mongoid doesn't implement `find_each` on `Mongoid::Criteria` apparently. Having read a bit about this I'm reasonably assured that the internals of `where().each` are using a cursor to iterate the collection. Further info here: http://stackoverflow.com/questions/7041224/finding-mongodb-records-in-batches-using-mongoid-ruby-adapter